runtime/sema/type.go

```diff
@@ -3366,6 +3366,22 @@ var AddressConversionFunctionType = &FunctionType{
 	},
 }
 
+const AddressTypeFromBytesFunctionName = "fromBytes"
+const AddressTypeFromBytesFunctionDocString = `
+Returns an Address from the given byte array
+`
+
+var AddressTypeFromBytesFunctionType = &FunctionType{
+	Parameters: []Parameter{
```

Comment:
Future flag (no actions needed now): Need to add purity annotations to this function type, once the stable-cadence branch is merged to master. Created issue #2466 to track this.